connectors/connectors.sh

```diff
@@ -50,13 +58,17 @@ get_connector_url() {
 
   # spark-bigquery
   if [[ $name == spark-bigquery ]]; then
-    # DATAPROC_VERSION is an environment variable set on the cluster.
+    # DATAPROC_IMAGE_VERSION is an environment variable set on the cluster.
     # We will use this to determine the appropriate connector to use
     # based on the scala version.
-    if [[ $(min_version "$DATAPROC_VERSION" 1.5) == 1.5 ]]; then
-      local -r scala_version=2.12
-    else
+    if compare_versions_lt $DATAPROC_IMAGE_VERSION 2.0 ; then
       local -r scala_version=2.11
+    else if compare_versions_lt $DATAPROC_IMAGE_VERSION 2.1 ; then
+           local -r scala_version=2.12
+         else if compare_versions_lt $DATAPROC_IMAGE_VERSION 2.2 ; then
```

Comment:
In the future, when a new Dataproc version is released, `scala_version` will be undefined. So, consider fail explicitly for unknown Dataproc version.

Reply:
done